### ISSUE TITLE
Support license in SPDX format

### ISSUE THREAD

**Issue by stitch** [internetstandards/Internet.nl-dashboard#100]
What does the implementation require? Todo: check out: https://spdx.org/

What files need to change (all files, the first lines?) etc...

**Comment by stitch** [comment]
Todo:
- [ ] Figure out if we need the SPDX tool to run each build (to assemble a bill of materials (SPDX file)).
- [ ] Find an SPDX tool that can be added to the build process that is written in something portable (aka: Self contained executable in Go, Python scripts etc, certainly NOT a java tool)
- [ ] Find if there is more to "bill of materials" than just an SPDX file and comments at the top of each file.
- [ ] Find out if the tool can add license information automatically where it is missing, as adding and especially maintaining it is tedious and error prone.
- [ ] Can PIP and NPM create SPDX files for all refered packages? Or do we still need a license tools website (one out of many).


My perspective on SPDX: Noble idea, poor execution. SPDX is simple, yet presented as complex as possible. End users don't need the 109 page specs document. They tell, they don't show.

Finding a tutorial was easy enough (https://github.com/david-a-wheeler/spdx-tutorial/), which reduces SPDX to this:
`file.py`
```
# Copyright [year file created] - [last year file modified], [project founder] and the [project name] contributors
# SPDX-License-Identifier: [SPDX license expression]
...
```

Probably a tool can be used to gather these files and create a .spdx file. My IDE (JetBrains PyCharm) has no native support for SPDX. Tooling in Java is not a first class candidate in build processes. (security, classpaths, bad qol, oracle, license).

**Comment by stitch** [comment]
SPDX is nothing more than adding the following two lines at the top of each file. This might even be done by a linter, which also adds the "last year file modified" automatically.

```
# Copyright 2020, [copyright holder] and the Internet.nl Dashboard contributors
# SPDX-License-Identifier: Apache-2.0
```

Solving this issue depends on #115.

**Comment by stitch** [comment]
Added SPDX-License-Identifier: Apache-2.0 to frontend and backend, copyright is addressed in another issue.
